RevenueCatUI/Data/Localization.swift

```diff
@@ -64,23 +64,24 @@ enum Localization {
 
     static func localized(
         packageType: PackageType,
-        locale: Locale = .current
+        locale: Locale,
```

Comment:
1. Defaulting to `.current` is hard to traceback for decision making and debugging to getting rid of it